.github/workflows/main.yml

```diff
@@ -0,0 +1,75 @@
+name: Pipeline
+
+on:
+  pull_request:
+    branches:
+      - develop
+    types: [opened, synchronize]
+  push:
+    branches:
+      - develop
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:latest
+        ports:
+          - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+        env:
+          POSTGRES_DB: ${{ secrets.DB_NAME }}
+          POSTGRES_USER: ${{ secrets.DB_USER }}
+          POSTGRES_PASSWORD: ${{ secrets.DB_PASSWORD }}
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.10.4'
```

Comment:
Any particular reason you are using python 3.10.4 here and another version in dockerfile?

Reply:
Changed only the basic python version similar to Dockerfile, because no alpine versions are available in github actions workflow.